### PR TITLE
unify and simplify project list

### PR DESCRIPTION
**Pull request by calvix**
towards https://github.com/giantswarm/giantswarm/issues/3473

this diff is super weird, sorry
basically I just moved the list into 3 vars for aws, azure and kvm and then they are reused ( and in one case we append extra component)

### CODE REVIEW

**Comment by calvix** [comment]
@JosephSalisbury WDYT ?

File: events/events.go [hunk 1]
```diff
@@ -108,256 +23,56 @@ var environmentProjects = map[Environment][]string{
...
-		"companyd",
-		"credentiald",
-		"desmotes",
-		"draughtsman",
-		"etcd-backup",
-		"g8s-cert-manager",
+	}
+	// aws project list + aws specific services
+	awsProjectList = append(baseProjectList,
 		"g8s-cloudwatch-exporter",
```
Comment (JosephSalisbury) [inline events/events.go]: The OCD side of me would alphabetise this

File: events/events.go [hunk 2]
```diff
@@ -108,256 +23,56 @@ var environmentProjects = map[Environment][]string{
...
+)
+
+// Environment is a name of an installation.
+type Environment string
+
+// environmentProjects is a mapping between Environments,
+// and the projects that should be deployed there.
+// We use codenames for installations to not expose customer information.
+var environmentProjects = map[Environment][]string{
+	"centaur": append(kvmProjectList, "pv-cleaner-operator"),
```
Comment (JosephSalisbury) [inline events/events.go]: Do we want to list these alphabetically too? Just to make it easier to find a specific installations.

Comment (calvix) [reply events/events.go]: oh Jesus the ordernacis

**Pull request merged**